integrations/hubspot/test/index.test.js

```diff
@@ -53,6 +55,12 @@ describe('HubSpot', function() {
         analytics.page();
         analytics.assert(window._hsq instanceof Array);
       });
+
+      it('should check for loadChatWidget to open chat widget', function() {
+        if (this.options.loadChatWidget) {
+          window.HubSpotConversations.widget.load();
```

Comment:
where is the assertion?

Reply:
Updated the test. and one more thing I found that we are checking for the method call.